makefile

```diff
@@ -1,13 +1,13 @@
 venv:
 	python3 -m venv .venv
-	@echo 'run `source .venv/bin/activate` to use virtualenv'
+	@echo 'run `. .venv/bin/activate` to use virtualenv'
```

Comment:
Would prefer to leave these as-is – I think it's more clear about the intent and less likely to be visually confusing given the following dotted directory name.  But if there's a shell that doesn't support `source`, I'd like to know.

Reply:
Sure, I have just reverted this aspect of the PR.

On further investigation I was using the dash shell, since I was on an Ubuntu virtual machine where ``bin/sh`` is symlinked to ``bin/dash``, where ``source`` is not supported as it is [apparently](https://stackoverflow.com/questions/11588583/is-the-shells-source-posix-standard) a 'bashism' & not part of POSIX. But it is certainly an atypical shell, so over the top to make changes to accommodate it.